File: python/googlesearch.py
import requests

task_params = {
    'target': 'google_search',
    'query': 'history',
    'parse': True,
    'geo': 'London,England,United Kingdom'
}

username = 'SPusername'
password = 'SPpassword'

response = requests.post(
    'https://scrape.smartproxy.com/v1/tasks',
    json = task_params,
    auth = (username, password)
)

print(response.text)

File: nodejs/googlesearch.js
const response = await fetch(
    'https://scraper-api.smartproxy.com/v2/scrape', {
        method: 'POST',
        body: {
            target: 'google_search',
            query: 'Adidas',
            parse: true,
            domain: 'com',
            page_from: 1,
            num_pages: 10,
            locale: 'en-us',
            geo: 'Detroit,Maine,United States',
            device_type: 'desktop',
            google_results_language: 'en'
        },
        headers: {
            'Content-Type': 'application/json',
            'Authorization': 'Basic AUTH'
        },
    }
).catch(error => console.log(error));

console.log(response)
